### PR TITLE
[ticket/16287] After installation an error message regarding statisti…

### PR DESCRIPTION
**Pull request by paul999**
…cs submission is given

The phpBB.com website required specific fields to be available for the
statistics. This change switched over to a new statistics page on
www.phpbb.com that uses form fields instead, and that returns JSON as value.

PHPBB3-16287

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16287

### CODE REVIEW

File: phpBB/adm/style/acp_help_phpbb.html [hunk 1]
```diff
@@ -54,11 +54,11 @@ <h2><i class="icon fa-bar-chart"></i>{L_SEND_STATISTICS}</h2>
 <form action="{U_COLLECT_STATS}" method="post" target="questionaire_result" id="questionnaire-form">
 	<fieldset>
 		<p class="submit-buttons">
-			<!-- BEGIN providers -->
-				<!-- BEGIN values -->
-					<input type="hidden" name="{providers.NAME}[{providers.values.KEY}]" value="{providers.values.VALUE}" />
-				<!-- END values -->
-			<!-- END providers -->
+			{% for providers in loops.providers %}
```
Comment (3D-I) [inline phpBB/adm/style/acp_help_phpbb.html]: `loops.` is necessary?

Comment (marc1706) [reply phpBB/adm/style/acp_help_phpbb.html]: No, but it's IMHO correct according to twig :P

Comment (3D-I) [reply phpBB/adm/style/acp_help_phpbb.html]: We are used without it, done in some PR of mine. :P

Comment (3D-I) [reply phpBB/adm/style/acp_help_phpbb.html]: https://github.com/phpbb/phpbb/pull/5560/files#diff-a2839ab7151761dc17f76d5b289b5005R3

**Pull request merged**